data-prepper-plugins/http-sink/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/HttpSinkServiceIT.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.sink;
```

Comment:
Can you move this IT under `org.opensearch.dataprepper.plugins.sink.http` as well?

You may need to add gradle configurations for this IT as well, like [this](https://github.com/opensearch-project/data-prepper/blob/42e274da3de6620a9ffa8b1e176a68b6d16463f4/data-prepper-plugins/s3-sink/build.gradle#L34-L63).

Reply:
Thanks for review @oeyh, Sure I shall move to that package